utility/src/utility/threading.cpp

```diff
@@ -1,13 +1,13 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-
 #include "threading.h"
 
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
 namespace nap
 {
     
-    TaskQueue::TaskQueue(std::uint32_t maxQueueItems) : mQueue(maxQueueItems)
+    TaskQueue::TaskQueue(unsigned int maxQueueItems) : mQueue(maxQueueItems)
```

Comment:
why be less explicit here? also: use nap/numeric

Reply:
I removed all the unsigned definitions and just used "int" here. I think it should be just an ordinary int argument.

Reply:
use int when accuracy is of no importance at all, ie: you know it will be in bounds, always. Use nap::uint32 etc. when you want to be explicit and show intention.